INSTALL_GUIDE_WINDOWS.md

```diff
@@ -214,7 +214,7 @@ The extracted AWS CDI SDK solution, cdi_proj.sln, contains three test applicatio
     - Select the **Windows Start button**.
     - Type **Environment**, and it will match with **Edit the system environment variables**. Select this suggested option.
     - In the resulting pop-up window, select **Environment Variables**.
-    - In the **System Variables** section, select the **Path** variable and click **New**.
+    - In the **System Variables** section, select the **Path** variable and click **Edit**, then click **New**.
```

Comment:
These changes to the documentation are not directly related to the original issue, but were discovered as part of the investigation.